src/063.js

```diff
@@ -0,0 +1,126 @@
+module.exports = {
+  description: 'Point features with added language properties suffixed with "_mbx_name" and "name_".',
+  specification_reference: 'https://github.com/mapbox/vector-tile-spec/blob/master/2.1/README.md',
+  validity: {
+    v1: true,
+    v2: true
+  },
+  json: {
+    layers: [
+      {
+        version: 2,
+        name: 'top',
+        features: [
+          {
+            id: 1,
+            tags: [
+              0, 0, // population: 20
+            ],
+            type: 1, // point
+            geometry: [ 9, 50, 32 ]
+          },
+          {
+            id: 2,
+            tags: [
+              0, 0, // population: 20
+            ],
+            type: 1, // point
+            geometry: [ 9, 51, 33 ]
+          }
+        ],
+        keys: [
+          'population'
+        ],
+        values: [
+          { int_value: 20 }
+        ],
+        extent: 4096
+      },
+      {
+        version: 2,
+        name: 'bottom',
+        features: [
+          {
+            id: 11,
+            tags: [
+              0, 0, // name: Germany
+              1, 0, // name_en: Germany
+              2, 1, // name_fr: Allemagne
+              3, 1, // _mbx_name_fr: Allemagne
+              4, 2, // _mbx_name_gr: Deutschland
```

Comment:
A minor thing, but it might be nice to make a fake BCP-47 that's more obviously fake (e.g., XX). I got slipped up by this.

Reply:
Oops, didn't mean for this to be a fake code. I can change to `_mbx_name_de`. I don't think we have test cases that care about fake BCP-47 codes in the tile data. Unless you can think of one?

Reply:
No actually - good point, no reason I know of for a fake code.